httpserver/rust/src/lib.rs

```diff
@@ -26,10 +26,29 @@ impl HttpResponse {
         let body = serde_json::to_string(&payload)
             .map_err(|e| RpcError::Ser(e.to_string()))?
             .into_bytes();
+        let mut header = HeaderMap::new();
+        header.insert("Content-Type".to_string(), vec!("application/json".to_string()));
         Ok(HttpResponse {
             body,
             status_code,
-            ..Default::default()
+            header
+        })
+    }
+    pub fn json_with_headers<T>(payload: T, status_code: u16, header: std::collections::HashMap<String, Vec<String>>) -> Result<HttpResponse, RpcError>
```

Comment:
This will show on the `docs.rs` documentation, let's add a doc comment here indicating that this will include the headers on the response object _and_ that the `Content-Type` header will automatically be included as it's a JSON response payload

Reply:
Done!